operation/fhir-operation-validate/src/test/resources/Patient.json

```diff
@@ -0,0 +1,174 @@
+{
+  "resourceType": "Patient",
```

Comment:
Is there anything special about this resource, or is it just a random Patient example?
Personally I prefer testing with minimal resources that have only the fields necessary for the specific test case.
I have a utility for creating those in the fhir-model tests jar.
Alternatively, if we don't want to introduce that dependency here, you could build a minimal Patient quite easilly (basically none of the fields are required for that type).

Reply:
This has been fixed. Removed Patient.json and built up Patient resource using Patient.builder() with minimal fields as suggested.